Automattic/jetpack
# Remove/submenu items of reader newdash masterbar link

Related to # pe7F0s-17X-p2

## Proposed changes:

* Removes submenu items from the masterbar newdash reader item. It is desired to remove this older toggle/popout menu for the reader and simply have the reader tab in this context redirect to its existing href (the wordpress.com reader).

BEFORE

![before-menu](https://github.com/Automattic/jetpack/assets/28742426/d8d3ae44-e281-406b-b50b-65679ae9736a)

AFTER

![after-menu](https://github.com/Automattic/jetpack/assets/28742426/21f87c70-10dc-4882-ae8a-806c36b3790e)


### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






* Setup a jetpack connected site.
* Enable the wpcom masterbar. This can be done in wp-admin at Jetpack -> Settings -> Writing (tab at top) -> WordPress.com Toolbar (at the very bottom of this section).
* Run this branch of Jetpack on the site. One way to do this is to install [Jetpack beta tester plugin](https://jetpack.com/download-jetpack-beta/) on the site. From here you can go to wp-admin and goto Jetpack -> Jetpack Beta -> Jetpack (click "manage") -> and type in the name of this branch in the "Search for a Feature Branch" input. Enable this branch.
* Reload the page, and test using the masterbar on this site. The "Reader" menu should always redirect to the WordPress.com reader and not open a popout menu.
* Smoke test these changes on wpcom simple and WoA. ([instructions in this comment](https://github.com/Automattic/jetpack/pull/32763#issuecomment-1699259658))
